src/platform/packages/shared/kbn-apm-synthtrace/src/lib/apm/client/apm_synthtrace_es_client/index.ts

```diff
@@ -25,7 +25,15 @@ export enum ComponentTemplateName {
   TracesApmRum = 'traces-apm.rum@custom',
   TracesApmSampled = 'traces-apm.sampled@custom',
 }
-export type ApmSynthtracePipelines = 'default' | 'otelToApm' | 'apmToOtel';
+export enum ApmSynthtracePipelineTypes {
+  Default = 'default',
+  OtelToApm = 'otelToApm',
+  ApmToOtel = 'apmToOtel',
```

Comment:
NIT/Question: Those namings look a bit misleading IMO. What does APM to OTel means? APM Server + EDOT (or OTel collector - or maybe the not EDOT is the o`telToApm`)? Same to OTel to APM 🤔 ?

Reply:
> @crespocarlos can you please confirm?

confirmed.

> But which one is EDOT in this case, for example (the second one maybe?), or we don't have it yet?

There is nothing specific for EDOT. What exists is a pipeline that outputs otel data, reproducing what traces exporter/APM server would do. The input could be either elastic APM, apmToOtel, or semconv otelToApm (this name could be improved).

Otel without APM Server has to flow through to the traces exporter, which will generate the same output.

Reply:
Thanks for explaining @crespocarlos!
@miloszmarcinkowski I saw you added comments to explain the pipelines, thank you! I am resolving this